UI/GovernmentWnd.cpp

```diff
@@ -1468,15 +1599,50 @@ void GovernmentWnd::Refresh() {
     m_main_panel->Refresh();
 }
 
+double GovernmentWnd::GetPolicyZoomFactor() {
+    switch (m_policy_size_buttons->CheckedButton()) {
+    case 0: return 1;
+    case 1: return 0.75;
+    case 2: return 0.5;
+    }
+
+    DebugLogger() << "GovernmentWnd::GetPolicyZoomFactor(): m_policy_buttons no button checked.";
+    return 1;
+}
+
+GG::Pt GovernmentWnd::GetPolicySlotSize() {
+    double zoom_factor = GetPolicyZoomFactor();
+    const GG::X slot_width = static_cast<GG::X>(SLOT_CONTROL_WIDTH * zoom_factor);
```

Comment:
`static_cast<GG::X>` seems odd... normally they can be created by calling the constructor.